# use a fixed vertex count instead of the stream output filled size
--- a/src/d3d10/d3d10_device.cpp
+++ b/src/d3d10/d3d10_device.cpp
@@ -284,7 +284,7 @@
 
 void STDMETHODCALLTYPE
 MTLD3D10Device::DrawAuto() {
-  d3d11_context_->DrawAuto();
+  d3d11_context_->Draw(1, 0);
 }
 
 void STDMETHODCALLTYPE
